CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.55.1
```

Comment:
This should be `newVersion`
We pick a version name whenever we feel ready to bump the version.

Reply:
Changed to `## newVersion`